core/drivers/stm32_uart.c

```diff
@@ -113,19 +113,13 @@ static void register_secure_uart(struct stm32_uart_pdata *pd __maybe_unused)
 {
 #ifndef CFG_STM32MP25
 	stm32mp_register_secure_periph_iomem(pd->base.pa);
```

Comment:
looking at our static mapping, I don't think we need this as well, don't you agree?

Reply:
ok